src/analyses/guard_bdd.h

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************\
+
+Module: Guard Data Structure
+
+Author: Romain Brenguier, romain.brenguier@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Guard Data Structure
+/// Implementation using BDDs
+
+#ifndef CPROVER_ANALYSES_GUARD_BDD_H
+#define CPROVER_ANALYSES_GUARD_BDD_H
+
+#include <iosfwd>
+#include <memory>
+
+#include <solvers/bdd/bdd.h>
+#include <util/make_unique.h>
+#include <util/std_expr.h>
+
+struct guard_bdd_managert
```

Comment:
I note that this is entirely unrelated to `guard_managert`. Shouldn't they at least share an interface?